Modules/CodeTrek/Resources/views/modals/sidebar-feedback-modal.blade.php

```diff
@@ -0,0 +1,86 @@
+<div class="feedback-modal modal fade" id="candidatefeedback{{ $codeTrekApplicant->id }}" tabindex="-1" z-index="1"
```

Comment:
Correct naming convention
```suggestion
<div class="feedback-modal modal fade" id="candidateFeedback{{ $codeTrekApplicant->id }}" tabindex="-1" z-index="1"
```